docs/install.md

```diff
@@ -42,13 +42,35 @@ To **run the extension in a Firefox Nightly** browser, you can run:
 npm run-script run -- -b nightly
 ```
 
+This will automatically create a fresh new user profile that will not persist
+between runs. This means the data will be lost every time.
+
+## Running the extension with a persistent profile
+
+To **run the extension with a profile that persists** between runs, you can
+create a new profile by browsing to `about:profiles`.
+
+Once you have a new profile created (no matter the location), you can tell jpm
+(via npm) to run using that profile _and_ not to copy the profile temporarily
+so that any changes (e.g. adding new entries) will be saved and persist:
+
+```sh
+npm run-script run -- -p /path/to/profile --no-copy
```

Comment:
Not sure if we want to shorten this to `npm run run` (to be consistent with line 76 below).

Reply:
Ah yes, I think we can use the "run" short-hand a bit more consistently.